frontend/src/plugins/router.js

```diff
@@ -77,7 +80,15 @@ export default new Router({
                             meta: {
                                 step: 1
                             },
-                            props: true
+                            props: true,
+                            beforeEnter: (to, from, next) => {
+                                if (store.state.isCurrentWebsite) {
```

Comment:
this probably won't work like it should be, since route hook can be called before data will be fetched